bin/nbb

```diff
@@ -344,6 +339,24 @@ find_centos_7_ami_ids() {
   done
 }
 
+shutdown_instances() {
+  aws ec2 describe-instances --filters "Name=instance-state-name,Values=running" | jq -c -r '.Reservations[] .Instances[] .InstanceId' > instance_list
+  echo "You are about to terminate the following instances:"
+  cat instance_list
+  read -r -p "Are you sure? [y/N] " response
+  if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]
+  then
+    IFS=$'\n'  # make newlines the separator
+    for i in `cat ./instance_list`; do
+      echo "Shutting down $i"
+      aws ec2 terminate-instances --instance-ids $i
```

Comment:
Suggestion: perhaps wrap the `aws ec2` call in a subshell function and/or background the process (maybe make note of its PID somewhere too?). **I'm worried that if AWS is slow, it could take hours upon hours to delete all your instances**, especially if you have a lot of them. In other words, I don't know if this command is syncrounous/blocking or not, and either way we can't gaurantee it'll *stay* that way in future releases.